### ISSUE TITLE
loss function problem

### ISSUE THREAD

**Issue by i4yyds** [batsa003/salgan#9]
Hello, may I ask why the generator uses binary classification cross entropy? I think the mean square error should be used, because the saliency prediction is not a binary classification problem, but a regression problem.

**Comment by ErolCitak** [comment]
@SoulCoderCN At the first, they considered MSE as a content loss. Then authors presented their reasons at their [paper's](https://arxiv.org/pdf/1701.01081.pdf) 4.th page. Also you can check this [slide](https://www.slideshare.net/xavigiro/salgan-visual-saliency-prediction-with-generative-adversarial-networks) to get information.
